src/components/EarnDashboard/components/ProviderDetails/OpportunityRow.tsx

```diff
@@ -7,9 +8,10 @@ import { Amount } from 'components/Amount/Amount'
 import { AssetCell } from 'components/StakingVaults/Cells'
 import { RawText, Text } from 'components/Text'
 import { bn, bnOrZero } from 'lib/bignumber/bignumber'
-import type {
-  LpEarnOpportunityType,
-  StakingEarnOpportunityType,
+import {
+  DefiProvider,
+  type LpEarnOpportunityType,
+  type StakingEarnOpportunityType,
```

Comment:
ocd nitpick - `import type` separate from `import`

Reply:
Fixed in [0c651d9](https://github.com/shapeshift/web/pull/7540/commits/0c651d9cc5e874e6804fcf2e1bdf11d71b0fe660)